ceph/ceph/templates/bin/_start_mon.sh.tpl

```diff
@@ -42,6 +42,16 @@ function get_mon_config {
       exit 1
   fi
 
+  IP=$(echo ${MONMAP_ADD} |awk '{print $NF}')
```

Comment:
$MONMAP_ADD is the full CLI to create the monmap with monmaptool. It'll look like "--add mon1 1.1.1.1 --add mon2 1.1.1.2. --add mon3 1.1.1.3"
$IP will always be equal to the latest Pod returned by `kubectl get pods`. Might be better to loop through the IP in $MONMAP_ADD.